server/opts.go

```diff
@@ -205,6 +205,7 @@ type Options struct {
 	JetStream             bool          `json:"jetstream"`
 	JetStreamMaxMemory    int64         `json:"-"`
 	JetStreamMaxStore     int64         `json:"-"`
+	JetStreamDomain       string        `json:"-"`
```

Comment:
When this value is set, should the Jetstream startup list it like in the example below

```
[74483] 2021/05/06 10:00:02.844380 [INF]     _ ___ _____ ___ _____ ___ ___   _   __  __
[74483] 2021/05/06 10:00:02.844388 [INF]  _ | | __|_   _/ __|_   _| _ \ __| /_\ |  \/  |
[74483] 2021/05/06 10:00:02.844390 [INF] | || | _|  | | \__ \ | | |   / _| / _ \| |\/| |
[74483] 2021/05/06 10:00:02.844392 [INF]  \__/|___| |_| |___/ |_| |_|_\___/_/ \_\_|  |_|
[74483] 2021/05/06 10:00:02.844393 [INF]
[74483] 2021/05/06 10:00:02.844395 [INF]          https://docs.nats.io/jetstream
[74483] 2021/05/06 10:00:02.844397 [INF]
[74483] 2021/05/06 10:00:02.844400 [INF] ---------------- JETSTREAM ----------------
[74483] 2021/05/06 10:00:02.844404 [INF]   Max Memory:      48.00 GB
[74483] 2021/05/06 10:00:02.844406 [INF]   Max Storage:     509.30 GB
[74483] 2021/05/06 10:00:02.844408 [INF]   Store Directory: "store_server_3/jetstream"
[74483] 2021/05/06 10:00:02.844408 [INF]   Jetstream Domain: hub , available with prefix $JS.hub.API
[74483] 2021/05/06 10:00:02.844409 [INF] -------------------------------------------
```